packages/api/src/external/fhir/shared/extensions/extension.ts

```diff
@@ -2,6 +2,7 @@ import { BASE_EXTENSION_URL } from "./base-extension";
 
 // TODO #712: create this extension
 export const DATA_SOURCE_EXTENSION_URL = `${BASE_EXTENSION_URL}/data-source.json`;
+export const DOC_ID_EXTENSION_URL = `${BASE_EXTENSION_URL}/doc-id-extension.json`;
```

Comment:
This already exists on the `doc-id-extension.ts` file, let's remove from there so its set in only one place?

Reply:
Oh, good catch. I didn't realize we had duplicate code there in API vs Core. I'll deprecate the ones on the API